features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml

```diff
@@ -42,6 +42,7 @@ repos:
     - name: libucxx
       sub_dir: python/libucxx
       args: {install: *rapids_build_backend_args}
+      depends: [ucxx]
```

Comment:
This should probably also include ucxx-python.

Reply:
Also done.